cwms_radar_api/src/main/java/cwms/radar/helpers/DatabaseHelpers.java

```diff
@@ -0,0 +1,20 @@
+package cwms.radar.helpers;
+
+import java.sql.CallableStatement;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class DatabaseHelpers {
```

Comment:
Actually didn't mean to push this one yet, but anyways. Should probably be the DSL, but the idea is the appropriate info will be present in the users principal object or session and this class would be responsible for knowing how to deal with that and just setup the session on the database side. eventually it would just be a passthrough and we'd handle authorization at the API level here.